static/js/compose.js

```diff
@@ -88,6 +88,12 @@ export function clear_preview_area() {
     $("#compose .markdown_preview").show();
 }
 
+export function collapse_compose_box() {
```

Comment:
I think this name is misleading, given that "collapse" might suggest closed.  I think probably cleanest would be to just to inline this code into the caller, since it's only 2 lines of code called from one place